frontend/src/components/Layout/PageFooter/index.tsx

```diff
@@ -16,6 +16,6 @@ export const PageFooter = () => (
     bgcolor="primary.main"
     color="primary.contrastText"
   >
-    <Typography noWrap>Footer</Typography>
+    <Typography noWrap>v{process.env.REACT_APP_VERSION}</Typography>
```

Comment:
i was hopping this could read the package.json file.

Reply:
I'd be interested to hear your reasoning on why doing it this way is necessary.

Reply:
@kshapka-bcgov so is there some automatic thing that happens, with this REACT_APP_VERSION, that picks up the version from the package.json... If there is, problem solved.  I just don't want to have to update this variable every time.

Reply:
From my PR description... This requires adding REACT_APP_VERSION=$npm_package_version to the frontend .env file in order to display version information. 😄 

However, I was under the impression react was a pain to import files from outside src, but it seems an update fixed that some time ago. I've changed it to just import the package.json file now.